app/Controllers/AdminDashboard.php

```diff
@@ -113,4 +114,52 @@ public function getAllEvents() {
         $events = $eventModel->getAll();
         return $this->response->setJSON($events);
     }
+
+    public function getEventSpeakers(int $eventId)
+    {
+        $eventModel = model(EventModel::class);
+        $events = $eventModel->getAll();
+        if (!in_array($eventId, array_column($events, 'id'))) {
+            return $this->response->setStatusCode(404);
+        }
+
+        $speakerModel = model(SpeakerModel::class);
+        $speakers = $speakerModel->getApproved($eventId);
```

Comment:
When a Speaker updates its Entry a new unapproved entry will be generated.
Do we set an old entry than and after approving the `visible_from` fields is the old value again?
Or will there be a function that updates the `visible_from` field when we approve it to the date that the last approved entry is holding?

Or do I miss something?